### PR TITLE
re-worded event from glossary

### LINKED ISSUE #39931: Unclear glossary entry for Event
This is the first line of `Event` in the glossary, I don't think it is clear:

> Each Event is a report of an event somewhere in the [cluster](https://kubernetes.io/docs/reference/glossary/?all=true#term-cluster).

### PR DESCRIPTION
**Pull request by KhushPatibandha**
resolved #39931

### CODE REVIEW

File: content/en/docs/reference/glossary/event.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ id: event
 date: 2022-01-16
 full_link: /docs/reference/kubernetes-api/cluster-resources/event-v1/
 short_description: >
-   A report of an event somewhere in the cluster. It generally denotes some state change in the system.
+   Refers to an object that describes a notable occurrence or change of state in the system.
```
Comment (NitishKumar06) [inline content/en/docs/reference/glossary/event.md:7]:
```suggestion
   Refers to an object that describes a notable occurrence or change of state in the system.
```
Refers what? Also, I'm not sure why we're adding an extra space. Can you explain if this was a typo?

**Review by NitishKumar06 — CHANGES_REQUESTED** [review]

File: content/en/docs/reference/glossary/event.md (hunk 1 above)
Comment (KhushPatibandha) [reply content/en/docs/reference/glossary/event.md:7]: Yes, I am have added a space while typing mb.

Comment (KhushPatibandha) [reply content/en/docs/reference/glossary/event.md:7]: > Refers what? Also, I'm not sure why we're adding an extra space. Can you explain if this was a typo?

"refers to an object" is not the correct way to put it?

**Comment by tengqm** [comment]
TBH, I failed to see the value out of this PR.
/hold

File: content/en/docs/reference/glossary/event.md [hunk 2]
```diff
@@ -4,13 +4,13 @@ id: event
...
```
Comment (divya-mohan0209) [inline content/en/docs/reference/glossary/event.md:7]: WDYT?
```suggestion
Events are Kubernetes objects that describe some state change in the system.
```

File: content/en/docs/reference/glossary/event.md [hunk 3]
```diff
@@ -4,13 +4,13 @@ id: event
...
 aka: 
 tags:
 - core-object
 - fundamental
 ---
-Each Event is a report of an event somewhere in the {{< glossary_tooltip text="cluster" term_id="cluster" >}}. 
+An Event refers to an object that describes a notable occurrence in the system.
```
Comment (divya-mohan0209) [inline content/en/docs/reference/glossary/event.md:13]: I think the suggested one is unnecessarily verbose.The event is a Kubernetes object that describes state change/notable occurrences in the system.

**Pull request merged**